client/verta/verta/_deployment/endpoint.py

```diff
@@ -9,26 +9,25 @@
 
 
 class Endpoint(object):
-    def __init__(self, conn, conf, workspace, id):
+    def __init__(self, conn, conf, workspace, endpoint_json):
```

Comment:
Small request: simply the `id` would be preferred for now, thanks!
```suggestion
    def __init__(self, conn, conf, workspace, id):
```